frontend/src/screens/wallet/send/ConfirmPayment.tsx

```diff
@@ -110,7 +110,7 @@ export default function ConfirmPayment() {
               />
             </div>
             {invoice.description && (
-              <p className="text-lg text-muted-foreground">
+              <p className="text-lg text-muted-foreground break-all">
```

Comment:
we should almost never have to use `break-all` 

Can you try `break-anywhere` instead?